spring-aop/src/main/java/org/springframework/aop/aspectj/AspectJAdviceParameterNameDiscoverer.java

```diff
@@ -272,22 +272,14 @@ public void setThrowingName(@Nullable String throwingName) {
 				}
 			}
 		}
-		catch (AmbiguousBindingException ambigEx) {
+		catch (AmbiguousBindingException | IllegalArgumentException ambigEx) {
```

Comment:
That's nice. Could you please consider polishing this suggestion a bit more by renaming the variable to `ex`? The only reason we're not using it here is because we have two catch blocks

Reply:
Sure, thanks!